lib/embulk/input/google_analytics/plugin.rb

```diff
@@ -215,11 +214,13 @@ def calculate_next_times(client_time_zone, fetched_latest_time)
             # no records fetched, don't modify config_diff
             task_report = {
               start_date: task["start_date"],
-              end_date: task["end_date"],
-              last_record_time: task["last_record_time"],
+              end_date: task["end_date"]
             }
+            # write last_record_time only when last_record_time is not nil and not empty
+            unless task["last_record_time"].nil? || task["last_record_time"].empty?
```

Comment:
```suggestion
            unless task["last_record_time"].blank?
```
It covers `nil` checking as well:
```
>> nil.blank?
=> true
```

Reply:
Done